nav_msgs/Path left_foot
nav_msgs/Path right_foot
walk_msgs/PathPoint3d center_of_mass
walk_msgs/PathPoint2d zmp
